src/main/UseCases/UseCaseUpdate.java

```diff
@@ -0,0 +1,72 @@
+import java.util.*;
+
+public class UseCaseUpdate {
```

Comment:
May be we can rename class to TempDataStorageUseCase for clarity as it's the use case interactor for our dictionary TempDataStorage